src/cli/run_command.zig

```diff
@@ -348,13 +349,23 @@ pub const RunCommand = struct {
             Output.flush();
         }
 
+        const envp = try env.map.createNullDelimitedEnvMap(allocator);
```

Comment:
on a different branch i had a version of this function that did one large allocation so that the defer free would be a single free.

code is https://github.com/oven-sh/bun/pull/9642 and might be cleaner to use.

it was put on pause because of change of focus to get 1.1 out.